src/controllers/commentContreller.js

```diff
@@ -0,0 +1,71 @@
+const { ConnectionReadyEvent } = require("typeorm");
+const { commentContreller } = require("../services");
+const { keyCheck } = require("../utils/keyCheck");
+
+
+const getUserByComment = async (req, res) => {
+        const user = req.headers.Authorization;
+        const getComment = await commentServies.getUserComment(user);
+
+        // 댓글이 있는 경우
+        if(!getComment)
+        return res.status(200).json({
+            message: 'READ_SUCCESS',
+            
+            
+        });
+    }
```

Comment:
```suggestion

const getCommentsByUser = async (req, res) => {
        const user = req.headers.Authorization;
        const getComment = await commentServies.getUserComment(user);

        // 댓글이 있는 경우
        if(!getComment)
        return res.status(200).json({
            message: 'READ_SUCCESS',
        });
    }
```
1. 의미가 명확한 함수 이름을 사용해 주세요.
2. 불필요한 공백은 제거해주세요.

Reply:
JavaScript는 indent 2칸이 기본입니다 너무... 넓어요...